misc/zsh-completion

```diff
@@ -31,42 +31,42 @@ __get_targets() {
   eval ${targets_command} 2>/dev/null | cut -d: -f1
 }
 
-__get_tools() {
-  ninja -t list 2>/dev/null | while read -r a b; do echo $a; done | tail -n +2
+(( $+functions[_ninja-get-tools] )) || _ninja-get-tools() {
+  ninja -t list 2> /dev/null | sed -e '1d;s/^ *//;s/ \+/:/'
```

Comment:
While for some people `sed` is a native language, most of us don't find ourselves fluent.

It would be a huge help to people looking at this file in the future if you could clearly document in the code what you're intended outcome is for each of these sed script calls.

Reply:
Here `-e '1d;s/^ *//;s/ \+/:/'` is equivalent to `-e '1d'` (remove the first line), `-e 's/^ *//'` (remove leading spaces), and `-e 's/ \+/:/'` (remove multiple spaces with colon)

Reply:
Sure. Put that in the file so someone looking at it in 3 years has a chance of understanding

Reply:
Done.